Firestore/Source/Model/FSTMutation.m

```diff
@@ -486,16 +492,26 @@ - (FSTMaybeDocument *_Nullable)applyTo:(FSTMaybeDocument *_Nullable)maybeDoc
  * Creates an array of "transform results" (a transform result is a field value representing the
  * result of applying a transform) for use when applying an FSTTransformMutation locally.
  *
+ * @param previousDocument The document prior to applying this mutation batch.
  * @param localWriteTime The local time of the transform mutation (used to generate
  * FSTServerTimestampValues).
  * @return The transform results array.
  */
-- (NSArray<FSTFieldValue *> *)localTransformResultsWithWriteTime:(FSTTimestamp *)localWriteTime {
+- (NSArray<FSTFieldValue *> *)
+localTransformResultsWithPreviousDocument:(FSTMaybeDocument *_Nullable)previousDocument
```

Comment:
Here you're calling this a "previous document" where elsewhere you're referring to it as the "base document". These are the same thing, right? If so you should name these concepts consistently.

Reply:
Renamed to `baseDocument`, which seems to be the better name in more places.